lib/commons/text/accessible-text.js

```diff
@@ -12,8 +12,8 @@ import { getNodeFromTree } from '../../core/utils';
  * @return {string}
  */
 function accessibleText(element, context) {
-  const virtualNode = getNodeFromTree(element); // throws an exception on purpose if axe._tree not correct
```

Comment:
The comment is not tree as `getNodeFromTree` does not throw, just returns `null` if the node isn't found.